vms/platformvm/txs/camino_address_state_tx.go

```diff
@@ -22,8 +22,10 @@ type (
 const (
 	// Bits
 
-	AddressStateBitRoleAdmin AddressStateBit = 0
-	AddressStateBitRoleKYC   AddressStateBit = 1
+	AddressStateBitRoleAdmin         AddressStateBit = 0
```

Comment:
In future we should postpone such cleanups - it makes it hard to review mixed content

Reply:
I think thats whitespace change
With whitespace turned off this lines are unchanged
No cleanup in this file here, only meaningful changes right on PRs topic :) (I think so)